src/utils/environmentUtils.ts

```diff
@@ -0,0 +1,3 @@
+export const getStaticFilePath = (path: string): string => {
+  return process.env.NODE_ENV === 'production' ? `/dashboard${path}` : path;
+};
```

Comment:
Not the smartest solution but could do the trick for now. You can give it a try if you want. 
I'm going to investigate about rewrite a little bit more any way. It is not ideal that we have to use this util for every single static file.